sherpa/astro/io/crates_backend.py

```diff
@@ -952,8 +952,7 @@ def get_pha_data(arg, make_copy=True, use_background=False):
         # Chandra Level 3 PHA files
         for ii in range(phadataset.get_ncrates()):
             block = phadataset.get_crate(ii + 1)
-            hduclas2 = block.get_key('HDUCLAS2')
-            if hduclas2 is not None and hduclas2.value == 'BKG':
+            if (_try_hdr_key(block, 'HDUCLAS2') == 'BKG'):
```

Comment:
And sure enough, the change in crates is coming right now. Crates should be returning strings soon :)
I asked that we have a CIAOX build with the crates changes but not the sherpa ones to compensate. I would like to see tests failing, and I'll probably add one test for this issue myself. Again, I want to see the new test fail before I push a fix for the next CIAOX build.